Assets/Plugins/GameJolt/Scripts/API/Settings.cs

```diff
@@ -20,6 +20,10 @@ public class Settings : ScriptableObject {
 		public bool autoPing = true;
 		[Tooltip("Cache High Score Tables and Trophies information for faster display.")]
 		public bool useCaching = true;
+		[Tooltip("The PlayerPrefs key for the user credentials if the 'Remember Me' option is ticked.")]
+		public string userCredentialsPreferences = "GJ-API-User-Credentials";
+		[Tooltip("The key used to encrypt the user credentials.")]
+		public string encryptionKey = "EnterSomeNicePassword";
```

Comment:
I'm afraid many people won't change this. Could the default be some random text?

Reply:
yeah, you're right, people are lazy 😄 
One simple solution would be to set the default password to empty string and then generate a new random password if the user opens the settings object and the current password is empty string.